tests/CommandLine.Tests/Unit/Text/HelpTextTests.cs

```diff
@@ -315,7 +315,7 @@ public void Invoke_AutoBuild_for_Options_returns_appropriate_formatted_text()
                     });
 
             // Exercize system
-            var helpText = HelpText.AutoBuild(fakeResult);
+            var helpText = HelpText.AutoBuild(fakeResult, new ParserSettings());

```

Comment:
All of these test updates should've alerted you to the fact that you broke the existing expected interface.  This will cause problems for people simply upgrading from a patch release, we'd have to make it wait until a full version release.

As noted elsewhere, you should preserve this method and make your new method an overload with the old method calling the new method with that default `ParserSettings` instance.